pkg/process/checks/net.go

```diff
@@ -33,6 +33,16 @@ type ConnectionsCheck struct {
 	tracerClientID         string
 	networkID              string
 	notInitializedLogLimit *procutil.LogLimit
+	lastTelemetry          telemetry
+}
+
+type telemetry struct {
+	KprobesTriggered          int64
+	KprobesMissed             int64
+	ConntrackRegisters        int64
+	ConntrackRegistersDropped int64
+	DnsPacketsProcessed       int64
+	ConnsClosed               int64
 }
```

Comment:
Can we use `model.CollectorConnectionsTelemetry` instead of creating a new type ?